File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.7 - Licznik znakow w tablicy.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int n, licznik = 0;
    cout << "Podaj liczbe z przedzialu (1 < n < 100): ";
    cin >> n;
    while (n < 1 || n > 100)
    {
        cout << "Podaj liczbe z przedzialu (1 < n < 100): ";
        cin >> n;
    }
    vector<char> znaki(n);
    for (int i = 0; i < n; i++)
    {
        cout << "Podaj znak: ";
        cin >> znaki[i];
    }
    for (int i = 0; i < n; i++)
    {
        if (znaki[i] == znaki[n - 1])
        {
            licznik++;
        }
    }
    cout << "Znak " << znaki[n - 1] << " znalazl sie w tablicy " << licznik << " razy." << endl;
    
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.7 - Licznik znakow w tablicy.py
przedzial = int(input("Podaj liczbe z przedzialu (0 < L < 100): "))

licznik = 0
tab = [' '] * przedzial

while przedzial < 0 or przedzial > 100:
    przedzial = int(input("Podaj liczbe z przedzialu (0 < L < 100): "))

for i in range(0, przedzial):
    tab[i] = str(input("Podaj znak: "))

for i in range(0, przedzial):
    if tab[przedzial - 1] == tab[i]:
        licznik += 1

print("Znak", tab[przedzial - 1], "znalazł się w tablicy", licznik, "razy.")